src/scripts/install.sh

```diff
@@ -1,42 +1,48 @@
 #!/bin/bash
-if echo "$OSTYPE "| grep darwin > /dev/null 2>&1; then
-    if [[ $EUID == 0 ]]; then export SUDO=""; else # Check if we're root
-        export SUDO="sudo";
-    fi
-    # shellcheck disable=SC2086
-    SC_DOWNLOAD_MAC="https://github.com/koalaman/shellcheck/releases/download/v${SC_INSTALL_VERSION}/shellcheck-v${SC_INSTALL_VERSION}.darwin.x86_64.tar.xz"
-    curl -LJO "$SC_DOWNLOAD_MAC"
+if [[ $EUID == 0 ]]; then export SUDO=""; else # Check if we're root
+    export SUDO="sudo"
+fi
+
+install_mac() {
+    curl -LJO "https://github.com/koalaman/shellcheck/releases/download/v${SC_INSTALL_VERSION}/shellcheck-v${SC_INSTALL_VERSION}.darwin.x86_64.tar.xz"
     tar -xvf "shellcheck-v$SC_INSTALL_VERSION.darwin.x86_64.tar.xz"
     cd "shellcheck-v$SC_INSTALL_VERSION/" || false
```

Comment:
I'm unsure what's the purpose behind `false` but since we don't exit anymore in the function, it might be better to replace it with a non-zero return. This way the application can exit properly. (Unless `false` does that already).

```suggestion
    cd "shellcheck-v$SC_INSTALL_VERSION/" || return 1
```